src/ops.rs

```diff
@@ -264,6 +264,12 @@ fn op_code_fetch(
       }
       _ => (),
     };
+    match out.maybe_source_map {
+      Some(ref source_map) => {
```

Comment:
i think this is
```
if let Some(ref source_map) = out.maybe_source_map {
    msg_args.source_map = Some(builder.create_string(source_map));
}
```

Reply:
I was following the pattern of the `output_code` above.  Are you suggesting that is also not as idiomatic?

Reply:
yes https://doc.rust-lang.org/rust-by-example/flow_control/if_let.html